source/view-model.js

```diff
@@ -4,14 +4,69 @@
 
 /*********************** View-Model **********************/
 "use strict";
+function GameInfo(game){
+    this.getFieldCells = function(){
+        return game.getFieldCells();
+    };
+
+    this.getCurrentStatus = function(){
+        let gameStatus = game.currentStatus;
+        let playerName = gameStatus == 'Tie' ? 'It' : game.getCurrentPlayerName();
+        return  playerName + ' is ' + gameStatus;
+    };
+};
+
+let gamePreset = {
+    players: [
+        {type : 'human', name : 'First', symbol : 'X'},
+        {type : 'computer', name : 'Computer', symbol : 'O'}
+    ],
+
+    setPlayer: function(key, value){
```

Comment:
**Error**: looks like this method will find a specified key in the first player.
**Suggestion**: fix the error (make the method change player's properties based on the player's name) and rename the method to make it reflect its purpose better.
**Expected result**: `setPlayerProperty(playerName, propertyName, propertyValue)`, where `playerName` allows the method to find a player by their name, `propertyName` specifies which property to change and `propertyValue` specifies a value for this property. In the future tasks this method will be updated to find players by their IDs.